### PR TITLE
3638 trailing wau mau

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

Opted to implement this for clickhouse first in this PR. The postgres querying needs to be done differently.

*Please describe.*  
- implements active user math for clickhouse only
*If this affects the frontend, include screenshots.*  

## Checklist

- [x] person endpoints
- [x] breakdown
- [x] clickhouse/postgres

### CODE REVIEW

File: frontend/src/lib/constants.tsx [hunk 1]
```diff
@@ -87,6 +87,30 @@ export const MATHS: Record<string, any> = {
         onProperty: false,
         type: EVENT_MATH_TYPE,
     },
+    weekly_active: {
```
Comment (EDsCODE) [inline frontend/src/lib/constants.tsx:90]: @paolodamico What's the best way to feature flag this?

Also, could use some suggestions on how to best convey the difference between weekly active, monthly active vs unique users. Unique users will count uniques dependent on the interval provided in the time settings. But weekly active and monthly active will strictly do as said disregarding the interval. Should I change it to somehow be dependent on interval setting too?

**Comment by EDsCODE** [comment]
@buwilliams Taking you as a reviewer to take a look at how functionality gets added to clickhouse. There's a bit of repetition that I still need to clean up but feel free to go through it!

File: frontend/src/lib/constants.tsx (hunk 1 above)
Comment (timgl) [reply frontend/src/lib/constants.tsx:90]: As we talked about, `preflight?.ee_enabled` + a feature flag.

I think the descriptions are okay for now (though if @paolodamico has better suggestions..), can iterate on this if we see people are confused

**Review by timgl — APPROVED** [review]
Looks good with exception of that feature flag that needs adding. I also wasn't able to figure out if it's correct with local data but if we merge it behind feature flags we can test it in prod

**Pull request merged**